cmd/kubeadm/app/cmd/upgrade/plan.go

```diff
@@ -27,15 +27,18 @@ import (
 	"github.com/lithammer/dedent"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
```

Comment:
```suggestion
	"github.com/spf13/pflag"
```

please make sure this is how pflag is imported in all files.